test/test.js

```diff
@@ -21289,9 +21357,13 @@
       }, 192);
 
       setTimeout(function() {
+        assert.strictEqual(callCount, 1);
```

Comment:
I'm not sure how an added assert changes the behavior of a failing test.
Can you explain a bit more what the change does?